packages/wonder-blocks-modal/src/components/__tests__/close-button.test.js

```diff
@@ -23,9 +19,11 @@ describe("CloseButton", () => {
     test("testId should be set in the Icon element", () => {
         // Arrange
         const wrapper = mount(
-            <ModalContext.Provider value={{closeModal: () => {}}}>
-                <CloseButton testId="modal-example-close" />,
-            </ModalContext.Provider>,
+            <div>
+                <ModalContext.Provider value={{closeModal: () => {}}}>
+                    <CloseButton testId="modal-example-close" />,
+                </ModalContext.Provider>
+            </div>,
```

Comment:
We were calling `unmountAll` in `beforeEach()` and this test was running last in this suite so we weren't calling `unmount` on it before.